pkg/provider/okta/okta.go

```diff
@@ -809,15 +809,15 @@ func verifyMfa(oc *Client, oktaOrgHost string, loginDetails *creds.LoginDetails,
 				body = updatedContext.challengeResponseBody
 
 			case "TIMEOUT":
-				fmt.Printf(" Timeout\n")
+				log.Printf(" Timeout\n")
 				return "", errors.New("User did not accept MFA in time")
 
 			case "REJECTED":
-				fmt.Printf(" Rejected\n")
+				log.Printf(" Rejected\n")
 				return "", errors.New("MFA rejected by user")
 
 			default:
-				fmt.Printf(" Error\n")
+				log.Printf(" Error\n")
```

Comment:
Perhaps this could be a `log.Error`

Reply:
Another great point; we use logrus in this file:
https://github.com/Versent/saml2aws/blob/302865c7e362604783069cf124a8f98aefccc72e/pkg/provider/okta/okta.go#L45

I would assume using `logger` everywhere would make more sense, rather than using the log one and losing the context that logrus provides.